tools/install.sh

```diff
@@ -72,38 +78,21 @@ else
     exit 2;
 fi
 
-if [[ -n $(which composer) ]]; then
-    echo ""
-    echo "PHP Composer appears to be installed."
-    composer_scr="composer install --no-dev"
-elif [[ -x ../composer ]]; then
-    echo ""
-    echo "PHP Composer appears to be installed."
-    composer_scr="./composer install --no-dev"
-else
-    echo ""
-    echo "PHP Composer does not appear to be installed. Please install it before running this script."
-    echo ""
-    echo "(e.g. wget https://getcomposer.org/installer"
-    echo "php installer --install-dir=/usr/local/bin --filename=composer)"
-    echo "while having root permission)";
+PHP_VERSION=$(php -v | head -n 1 | cut -d " " -f 2 | cut -f1-2 -d".")
+if [ "`echo "${PHP_VERSION} < 7.2 " | bc`" -eq 0  ]; then
+    echo "Your PHP version is too low to install Loris."
+    echo "Please update your PHP to 7.2 or higher. "
     exit 2;
 fi
 
 echo ""
 
 cat <<QUESTIONS
-This install script will ask you to provide inputs for different steps.
-Please ensure you have the following information ready (if applicable):
-
-  1) Your project directory name.
-     (Will be used to modify the paths for Imaging data in the generated
-     config.xml file for LORIS, and may also be used to automatically
-     create/install apache config files.) If unsure, a default like "LORIS"
-     should be acceptable.
+This install script will ask you to provide your project name. 
+If unsure, we suggest using "LORIS"
+(This will be used to define Imaging data filepaths automatically in a config.xml file for LORIS, and (optionally) for the apache configuration file.) 
```

Comment:
I'm pretty sure this statement could be changed to "This will be used IFF you want this script to configure your apache"